packages/connect-examples/new-example/app/components/ui/Label.tsx

```
@@ -0,0 +1,19 @@
+interface LabelProps extends React.LabelHTMLAttributes<HTMLLabelElement> {
+  className?: string;
+}
+
+const Label = React.forwardRef<HTMLLabelElement, LabelProps>(
+  ({ className = "", ...props }, ref) => (
+    // eslint-disable-next-line jsx-a11y/label-has-associated-control
+    <label
+      ref={ref}
+      className={`text-sm font-medium leading-none peer-disabled:cursor-not-allowed peer-disabled:opacity-70 ${className}`}
+      {...props}
+    />
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding an accessible text content to the label element.
Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)